### DISCUSSION TITLE
Forwarding traffic to my local website?

### DISCUSSION THREAD

**Discussion by Sagarshar06** [telekom-security/tpotce#1574] [Q&A]
I want to forward traffic to my website and want to configure it on TPOT port 443 to view the gain insights about web attacks. How can I achieve it?

**Comment by Ayaan7345** [comment]
To achieve this go to your DNS settings on your hosting provider and create a SVR record with the ip of the tpot and port 64297. If you have kept the default settings of the tpot this will work.
Is there a specific reason you want change the tpot webserver port to 443?

**Reply by Sagarshar06** [reply]
So, to be clear I have hosted it in Azure, and I have created a website to lure attackers and gain insights. Do I have to change the DNS setting in Azure or something needs to be configured in TPOT?

**Reply by Ayaan7345** [reply]
Are the TPOT and the website the same machine? if so you could do the normal install of the Tpot and then turn of the citrix honeypot that is using port 443 so you could make space for your webserver. 

If you could send a picture of the network topology it would help alot.

**Reply by Sagarshar06** [reply]
There should be a connectivity between TPOT and Website on port 443, right? How can I achieve the connectivity and turn off Citrix honeypot? 
Yes, I am planning on deploying the webserver on the same machine where TPOT is installed.

**Reply by Ayaan7345** [reply]
To turn off specific honeypots go to #1416. What do you you mean when you say connectivity? To the server or connecting the website to the honeypot?

**Reply by Sagarshar06** [reply]
I have created a dummy website and want to run it as a service in tpot on port 443. This website should act as honeytrap so that I can get the insights of the attacks which takes place on Web Application.

**Reply by Ayaan7345** [reply]
Ok yeah all you have to do then is when you install TPOT make sure to comment out the Citrix honeypot. Then run your honeypot and you will be ok. There is no additional networking you have to do because port 443 will be open for your honeytrap.

**Reply by Sagarshar06** [reply]
In which yml file should I make the changes to comment out Citrix Honeypot in tpot.yml or docker-compose.yml?
